routine-data.h

```diff
@@ -0,0 +1,29 @@
+#ifndef __ROUTINE_DATA__
+#define __ROUTINE_DATA__
+
+void gatherData() {  
+  int d = getMedianData();
+  publishInt("data", d);
+  data[current_sample++] = d;
+}
+
+int processData() {
```

Comment:
What about more general approach.
For example - passing pointer to function and then call specified function by pointer?
Because here we need getMedianData function but if we needed another one?

Reply:
function by pointer is great approach but hardly spplieble here - finally we'll have only 1 function - smth like getADCData

as for filter data - we'll apply all of available filters to gatherd data and send filtering results to different topics

but in another pull request